tensorflow/tools/api/tests/module_test.py

```diff
@@ -74,11 +74,10 @@ def testSummaryMerged(self):
     # If we use v2 API, check for create_file_writer,
     # otherwise check for FileWriter.
     if hasattr(tf, '_major_api_version') and tf._major_api_version == 2:
-      tf.summary.create_file_writer
+      tf.compat.v2.summary.create_file_writer
```

Comment:
`tf.compat.v2` is TF 2.x but then `tf.` is the same API. Do we really need to change this line?

Reply:
Required changes made.